lib/node_modules/@stdlib/math/base/special/cpolar/src/addon.c

```diff
@@ -0,0 +1,140 @@
+		status = napi_throw_error( env, NULL, "invalid invocation. Insufficient arguments." );
+		assert( status == napi_ok );
+		return NULL;
+	}
+	if ( argc > 2 ) {
+		status = napi_throw_error( env, NULL, "invalid invocation. Too many arguments." );
+		assert( status == napi_ok );
+		return NULL;
+	}
+
+    bool res;
+	status = napi_is_typedarray( env, argv[ 0 ], &res );
+	assert( status == napi_ok );
+	if ( res == false ) {
+		status = napi_throw_type_error( env, NULL, "invalid argument. First argument must be a Float64Array." );
+		assert( status == napi_ok );
+		return NULL;
+	}
+
+    // Get the first element out
+    napi_typedarray_type vtype0;
+	size_t len;
+	void *Out;
+	status = napi_get_typedarray_info( env, argv[ 0 ], &vtype0, &len, &Out, NULL, NULL );
+	assert( status == napi_ok );
+	if ( vtype0 != napi_float64_array ) {
+		status = napi_throw_type_error( env, NULL, "invalid argument. First argument must be a Float64Array." );
+		assert( status == napi_ok );
+		return NULL;
+	}
+	if ( len != 2 ) {
+		status = napi_throw_range_error( env, NULL, "invalid argument. First argument must have 2 elements." );
+		assert( status == napi_ok );
+		return NULL;
+	}
+
+    // Get the real component
+    napi_value xre;
+	status = napi_get_named_property( env, argv[ 1 ], "re", &xre );
+	assert( status == napi_ok );
+
+	napi_valuetype xretype;
+	status = napi_typeof( env, xre, &xretype );
+	assert( status == napi_ok );
+	if ( xretype != napi_number ) {
+		status = napi_throw_type_error( env, NULL, "invalid argument. First argument must have a real component which is a number." );
+		assert( status == napi_ok );
+		return NULL;
+	}
+
+    // Get the imaginary component
+	napi_value xim;
+	status = napi_get_named_property( env, argv[ 1 ], "im", &xim );
+	assert( status == napi_ok );
+
+	napi_valuetype ximtype;
+	status = napi_typeof( env, xim, &ximtype );
+	assert( status == napi_ok );
+	if ( ximtype != napi_number ) {
+		status = napi_throw_type_error( env, NULL, "invalid argument. First argument must have an imaginary component which a number." );
+		assert( status == napi_ok );
+		return NULL;
+	}
+
+    double re;
+	status = napi_get_value_double( env, xre, &re );
+	assert( status == napi_ok );
+
+	double im;
+	status = napi_get_value_double( env, xim, &im );
+	assert( status == napi_ok );
+
+    double abs;
+	double cphase;
+    stdlib_base_cpolar( stdlib_complex128( re, im ), &abs, &cphase );
+
+    double *op = (double *)Out;
+	op[ 0 ] = abs;
+	op[ 1 ] = cphase;
+
+    return NULL;
+}
+
+/**
+* Initializes a Node-API module.
+*
+* @private
+* @param env      environment under which the function is invoked
+* @param exports  exports object
+* @return         main export
+*/
+static napi_value init( napi_env env, napi_value exports ) {
+	napi_value fcn;
+	napi_status status = napi_create_function( env, "exports", NAPI_AUTO_LENGTH, addon, NULL, &fcn );
+	assert( status == napi_ok );
+	return fcn;
+}
+
+NAPI_MODULE( NODE_GYP_MODULE_NAME, init )
```

Comment:
```suggestion
NAPI_MODULE( NODE_GYP_MODULE_NAME, init )

```

Missing trailing newline.